internal/indexmanagement/scripts.go

```diff
@@ -415,9 +415,11 @@ function getWriteAliases() {
   # "_cat/aliases/${policy}*-write?h=alias" | uniq
   aliasResponse="$(catWriteAliases "$policy")"
 
-  if [ -z "$aliasResponse" ]; then
+  if [ "$aliasResponse" = "error" ]; then
```

Comment:
I re-read the parts about bash comparisons again and I think this should be more correct:

```suggestion
  if [[ "$aliasResponse" == "error" ]]; then
```